src/sources/aws_sqs/events.rs

```diff
@@ -8,15 +8,12 @@ pub struct AwsSqsBytesReceived {
 }
 
 impl InternalEvent for AwsSqsBytesReceived {
```

Comment:
Not related to your PR, but I'm curious why these are here and not in `src/internal_events`.

Reply:
Hmm, yeah, that is odd.

Reply:
to be honest, I know it's the way to put them all in `src/internal_events`, but considering it's only used in this file, I don't see that as a problem 🤔

Reply:
I'd still move them over. We have other examples of events that are only used in one place that are in `src/internal_events` and it's nice to be able to just look there for event definitions.